### ISSUE TITLE
Failed to replace entity Warnings in KC 22

### ISSUE THREAD

**Issue by souravs17031999** [keycloak/keycloak#23634]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

infinispan

### Describe the bug

Hi, 

We are seeing lots of warnings as shown below in our production environment when we upgraded to KC 22.0.1:

```
WARN  [org.keycloak.models.sessions.infinispan.changes.InfinispanChangelogBasedTransaction] (executor-thread-1285) Failed to replace entity '<id>' in cache 'sessions'
```

Also, sometimes warnings are like 

```
DEBUG [org.keycloak.models.sessions.infinispan.changes.InfinispanChangelogBasedTransaction] (executor-thread-65247) Replace failed for entity: <id_1>, old version <id_2>, new version <id_3>. Will try again
```

### Version

22.0.1

### Expected behavior

No Warnings should be seen 

### Actual behavior

Lots of these warnings are being seen increasingly as we upgraded our cluster to KC 22.
Also, pod memory keeps increasing as soon as pods are restarted, can it be related ?

### How to Reproduce?

Start KC server in K8S cluster (stateful sets) with 2 pods with version 22.0.1

**Comment by souravs17031999** [comment]
@mposolda @pedroigor @edewit @ahus1 could you please suggest any ideas where to look ?

**Comment by martin-kanis** [comment]
@souravs17031999 Thanks for reporting this. However, I'm unable to reproduce it with the steps you provided, with the latest nightly image. I ran cluster of 3 Keycloak nodes on Openshift using `keycloak-benchmark` project.  Then I executed:

1. AuthorizationCode scenario (login, logouts) for 300s
2. refresh token scenario for 300s
3. Client Credentials Grant scenario for 300s

No warnings seen in the logs. 

We will need more detailed steps to reproduce this.
